### PR TITLE
[14.0][ADD] sale_product_rating_verified

### PR DESCRIPTION
**Pull request by david-popa**
Verify if a user has previously bought a product

### CODE REVIEW

File: sale_product_rating_verified/models/rating.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+        for record in self:
+            record.purchase_verified = False
+            purchased_product = (
+                partner_product_count.get((record.partner_id.id, record.res_id), 0) > 0
+            )
+            if record.res_model == "product.product" and purchased_product:
+                record.purchase_verified = True
+
+    purchase_verified = fields.Boolean(
```
Comment (rousseldenis) [inline sale_product_rating_verified/models/rating.py]: Could you put fields definition on top ?

File: sale_product_rating_verified/models/rating.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        required=True, default=False, compute="_compute_verified", store=True
```
Comment (rousseldenis) [inline sale_product_rating_verified/models/rating.py]: I would have done _compute_purchase_verified()

**Review by rousseldenis — CHANGES_REQUESTED** [review]
LGTM. Could you maybe add a USAGE fragment with a screenshot ?

**Review by rousseldenis — APPROVED** [review]

**Review by gurneyalex — APPROVED** [review]
looks a bit strange that the "verified" badge does not change when the invoice gets paid, or that the date of the invoice is before the advice. This could be mentionned in the readme, I think.

**Pull request merged**